src/index.ts

```diff
@@ -19,16 +24,29 @@ interface RequiredAccountMethods {
 	cairoVersion?: CairoVersion
 	getCairoVersion?(classHash?: string): Promise<CairoVersion>
 	isWalnutLogsAdded?: boolean
-	provider?: { nodeUrl: string }
 }
 
+let isSentryInitialized = false
+
 async function sendLog(apiKey: string, account: RequiredAccountMethods, calls: AllowArray<Call>, transactionsDetail?: TransactionsDetail) {
+	if (!isSentryInitialized) {
+		Sentry.init({
+			dsn: SENTRY_DSN,
+			autoSessionTracking: false,
+			sendClientReports: false,
+			defaultIntegrations: false,
+			release: '1.1.11',
```

Comment:
How can we ensure that it gets updated with every new release?

Reply:
Made changes; now it is set to the package version